src/subcmd/withdraw_fees.rs

```diff
@@ -0,0 +1,68 @@
+use base64::{engine::general_purpose, Engine};
+use borsh::ser::BorshSerialize;
+use clap::Args;
+use solana_sdk::pubkey::Pubkey;
+use spl_governance::state::proposal_transaction::{AccountMetaData, InstructionData};
+use stakedex_interface::{WithdrawFeesIxArgs, WithdrawFeesKeys};
+use stakedex_sdk_common::find_fee_token_acc;
+
+use super::SubcmdExec;
+
+#[derive(Args, Debug)]
+#[command(
+    long_about = "Outputs a base64-encoded transaction that withdraws fees to the specified token account when signed by the admin authority."
+)]
+pub struct WithdrawFeesArgs {
+    #[arg(help = "Mint of the token to withdraw fees for.")]
+    pub mint: Pubkey,
+    #[arg(
+        help = "Destination token account to withdraw fees to. Defaults to the admin authority's associated token account if not provided."
+    )]
+    pub destination: Option<Pubkey>,
+}
+
+const ADMIN_AUTHORITY: &str = "A7jn1BA6LPHX8Wcmc8t476gjoLCG4PZakww19ZXfFRjX";
```

Comment:
sorry, could you just add this bit of doc in too? Thanks

```suggestion
/// TODO: import from stakedex crate when onchain program is made public
const ADMIN_AUTHORITY: &str = "A7jn1BA6LPHX8Wcmc8t476gjoLCG4PZakww19ZXfFRjX";
```